applications/istio/bookinfo/README.md

```diff
@@ -0,0 +1,130 @@
+# Istio Bookinfo demo in OAM way
+This application demonstrates how OAM bootstraps [Istio Bookinfo application](https://istio.io/latest/docs/examples/bookinfo/) with workload `Gateway`, `VirtualService`,
+`DestinationRule` deployed by OAM.
+
+## Prerequisites
+- Helm v3
+
+- OAM Kubernetes Runtime
+
+```
+kubectl create namespace crossplane-system
+helm repo add crossplane-alpha https://charts.crossplane.io/alpha
+helm install addon-oam-kubernetes-local --namespace crossplane-system crossplane-alpha/oam-core-resources
+```
+
+- OAM workloads/traits definitions
+
+```
+$ kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/release-0.12/docs/snippets/run/definitions.yaml
+workloaddefinition.core.oam.dev/containerizedworkloads.core.oam.dev created
+traitdefinition.core.oam.dev/manualscalertraits.core.oam.dev created
+workloaddefinition.core.oam.dev/postgresqlinstancerequirements.database.example.org created
+
+```
+
+(TODO)  ** The root cause why this depends on installation of Crossplane has to be investigated.**
+```
+$ kubectl apply -f https://raw.githubusercontent.com/crossplane/crossplane/release-0.12/docs/snippets/run/definitions.yaml
+
+unable to recognize "https://raw.githubusercontent.com/crossplane/crossplane/release-0.12/docs/snippets/run/definitions.yaml": no matches for kind "WorkloadDefinition" in version "core.oam.dev/v1alpha2"
+unable to recognize "https://raw.githubusercontent.com/crossplane/crossplane/release-0.12/docs/snippets/run/definitions.yaml": no matches for kind "TraitDefinition" in version "core.oam.dev/v1alpha2"
+unable to recognize "https://raw.githubusercontent.com/crossplane/crossplane/release-0.12/docs/snippets/run/definitions.yaml": no matches for kind "WorkloadDefinition" in version "core.oam.dev/v1alpha2"
+```
+
+- Istio 1.6.3
+
+Pleaser refer to [Download and Install Istio](https://istio.io/latest/docs/setup/getting-started/).
```

Comment:
add "And do not deploy the book info sample from the istio installation folder".